experiments/Markov/src/Markov.jl

```diff
@@ -1,5 +1,7 @@
 module Markov
 
 include("MarkovCategories.jl")
+include("StochMaps.jl")
```

Comment:
Should this line be here? I only see `StochExpr.jl`.

Reply:
Leftover from the last deletion.